nubia/internal/plugin_interface.py

```diff
@@ -137,9 +141,9 @@ def getBlacklistPlugin(self):
         """
         return CommandBlacklist()
 
-    def get_prompt_tokens(self):
+    def get_prompt_tokens(self) -> List[Tuple]:
```

Comment:
Let's refine this annotation to reflect the tuple elements' types.
Since pygments [does not expose](
https://bitbucket.org/birkenfeld/pygments-main/src/7941677dc77d4f2bf0bbd6140ade85a9454b8b80/pygments/token.py?at=default&fileviewer=file-view-default#token.py-13) the type of `Token.Username`, I think that `List[Tuple[Any, str]]` will suffice for now.